conjure-java-jackson-serialization/src/test/java/com/palantir/conjure/java/serialization/ObjectMappersTest.java

```diff
@@ -421,6 +423,39 @@ private void testTypeFactory(ObjectMapper mapper) {
         assertThat(mapper.getTypeFactory()).isInstanceOf(CaffeineCachingTypeFactory.class);
     }
 
+    @Test
+    public void testTypeFactoryCacheMetrics() throws IOException {
+        TaggedMetricRegistry registry = SharedTaggedMetricRegistries.getSingleton();
+        JsonDatabindTypefactoryCacheMetrics metrics = JsonDatabindTypefactoryCacheMetrics.of(registry);
+        Meter hit = metrics.hit();
+        Meter miss = metrics.miss();
+
+        ObjectMapper mapper = ObjectMappers.newServerJsonMapper();
+        long hitBefore = hit.getCount();
+        long missBefore = miss.getCount();
+
+        mapper.writeValue(ByteStreams.nullOutputStream(), new SimpleSerializable());
+
+        long hits = hit.getCount() - hitBefore;
+        long misses = miss.getCount() - missBefore;
+
+        assertThat(hits).isGreaterThan(0);
```

Comment:
should this be 0 hits?

```suggestion
        assertThat(hits).isZero();
```

Reply:
Internally this ends up around 3 due to various components mapping the same type. It's probably best not to make any asserts at all at this point, then assert the value has increased in the second round

Reply:
I'll update the test to operate directly on the TypeFactory since the ObjectMapper complicates things